packages/maskbook/src/plugins/Trader/services.ts

```diff
@@ -54,6 +59,8 @@ currentTradeProviderSettings.addListener((tradeProvier: TradeProvider) => {
             break
         case NetworkType.Arbitrum:
             throw new Error('TODO')
+        case NetworkType.xDai:
+            throw new Error('TODO')
```

Comment:
Seems like there are still some TODOs here. Is that going to be resolved in this PR?